org.eclipse.xtext.xbase/src-gen/org/eclipse/xtext/xbase/annotations/services/XbaseWithAnnotationsGrammarAccess.java

```diff
@@ -241,16 +244,16 @@
 		//']'
 		public Keyword getRightSquareBracketKeyword_0_2() { return cRightSquareBracketKeyword_0_2; }
 		
-		//XAnnotationOrExpression ({xbase::XListLiteral.elements+=current} (',' elements+=XAnnotationOrExpression)+)?
+		//XAnnotationOrExpression ({xbase :: XListLiteral.elements+= current} (',' elements+=XAnnotationOrExpression)+)?
 		public Group getGroup_1() { return cGroup_1; }
 		
 		//XAnnotationOrExpression
 		public RuleCall getXAnnotationOrExpressionParserRuleCall_1_0() { return cXAnnotationOrExpressionParserRuleCall_1_0; }
 		
-		//({xbase::XListLiteral.elements+=current} (',' elements+=XAnnotationOrExpression)+)?
+		//({xbase :: XListLiteral.elements+= current} (',' elements+=XAnnotationOrExpression)+)?
```

Comment:
I think I mentioned it before: Unexpected space after the assignment operator.

Reply:
Missing recursion, will be fixed with next commit.